doc/akri_connector/minimal-example-connector-metadata.json

```diff
@@ -0,0 +1,33 @@
+{
+  "aioConnectorMetadataSchemaVersion": "1.0.0",
+  "name": "RestConnector",
+  "description": "Connector for polling a REST server for information",
+  "version": "1.0.0",
+  "imageConfigurationSettings": {
+    "containerRegistryName": "contoso-acr.io",
+    "imageName": "RestConnector",
+    "tag": "1.0.0"
+  },
+  "inboundEndpoints": [
+    {
+      "name": "RestInboundEndpoint",
+      "version": "1.0",
+      "configSchemas": {
+        "datasetConfigSchema": {
+          "schema": {
+            "$schema": "http://json-schema.org/draft-07/schema#",
+            "title": "Dataset Config Schema",
+            "type": "object",
+            "properties": {
+              "SamplingInterval": {
+                "description": "How frequently to sample each dataset by default (in milliseconds)",
+                "type": "integer"
+              }
+            }
```

Comment:
you want to define a default value and maybe min, max. Integer: what happens if the user puts in a negative number, what is the connector doing when 0 is entered?

Reply:
This doc describes a fictional connector. I'd rather not get into that level of detail.